doc/source/conf.py

```diff
@@ -161,6 +161,7 @@
     'gallery_dirs': 'auto_examples',
     'backreferences_dir': 'api',
     'reference_url': {'skimage': None},
+    'thumbnail_size': (280, 196),  # Default CSS does 0.4 scaling (160, 112)
```

Comment:
is the final image size 160, 112?

what is the default thumbnail size? can this be added as a comment.

And your pagesize improvement would go a long way too.